client/router.js

```diff
@@ -369,3 +427,45 @@ Router.prototype._updateCallbacks = function () {
 
 Router.prototype._page = page;
 Router.prototype._qs = qs;
+
+function shouldCallTrigger(current, fn) {
+  var name = current.route.name;
+
+  if (typeof fn !== 'function') {
+    return false;
+  }
+
+  if (!name) {
```

Comment:
How should we declare `only` for this case? `only: ['']`, `only:[]` or something else?